src/core/server/saved_objects/migrationsv2/actions/integration_tests/actions.test.ts

```diff
@@ -1496,7 +1503,7 @@ describe('migration actions', () => {
                 }
               `);
     });
-    it('rejects if there are errors', async () => {
+    it('resolves left if there are write_block errors', async () => {
```

Comment:
I wanted to also add a IT test to assert that it still rejects for other errors, but it seems that using an non-existing index surprisingly leads to a timeout (which is handled by `catchRetryableEsClientErrors`) instead of a more final error, and I couldn't find a way to trigger another kind of error from ES.

If anyone has an idea, I'll take it. Else it's probably fine as it's covered in unit tests anyway.